src/xenserver/opt/xensource/www/XCP-ng-index.html

```diff
@@ -164,121 +162,280 @@
+					<p><a href="https://github.com/xcp-ng/xenadmin/releases/latest" class="btn">Download</a></p>
+				</div>
+			</div>
+		</div>
+	</div>
 	<hr>
-		<a href="https://xcp-ng.org/forum" class="btn">Community forum</a>
-</section>
-<!-- Modalcontent -->
-<div id="ex1" class="modal">
-  <p>SSH to your XCP-ng machine and execute the following:</p>
-    <code id="p1" style="color: #808080; background-color:#d9d9d9")> bash -c "$(curl -s http://xoa.io/deploy)"</code>
-        <button class="btn" onclick="copyToClipboard('#p1')"><i class="fas fa-paste"></i></button>
-  <p style="font-style: italic">This will automatically download/import/start the XOA appliance.</p>
-  <br>
-  <a href="#" rel="modal:close">Close</a>
+	<div class="section">
+		<h3 class="subtitle">Pro Support</h3>
+		<p>XCP-ng is a high performance enterprise level hypervisor, responding to the most demanding company activities from SMBs to Datacenters. We have your back when it comes to pro support. Visit our dedicated website to learn more.</p>
+		<div class="row">
+			<div class="col-xs">
+				<div class="support">
+					<a href="https://xcp-ng.com?pk_campaign=xcp-ng-html"><i class="fas fa-user-headset"></i></a>
+					<p class="description">Pro Support 24/7/365</p>	
+				</div>
+			</div>
+			<div class="col-xs">
+				<div class="support">
+					<a href="https://xcp-ng.com?pk_campaign=xcp-ng-html"><i class="fas fa-terminal"></i></a>
+					<p class="description">SSH remote support</p>		
+				</div>
+			</div>
+			<div class="col-xs">
+				<div class="support">
+					<a href="https://xcp-ng.com?pk_campaign=xcp-ng-html"><i class="fas fa-wrench"></i></a>
+						<p class="description">Setup assistance</p>	
+				</div>
+			</div>
+		</div>
+		<p><a href="https://xcp-ng.com?pk_campaign=xcp-ng-html" class="btn">Go Pro</a></p>
+	</div>
+	<hr>
+	<div class="section">
+		<h3 class="subtitle">Join our community</h3>
+		<p class="description">Thousands of people are using XCP-ng, hundreds are contributing on our community forum. Become a part of our community!</p>
+		<p><a href="https://xcp-ng.org/forum" class="btn">Community forum</a></p>
+	</div>
+	<!-- Modalcontent -->
+	<div id="ex1" class="modal">
+		<h3 class="subtitle">XOA quick deploy</h3>
+		<form id="connect" onsubmit="event.preventDefault(); connect()">
+			<fieldset>
+			  <p>
+				<label>
+				  Host username:<br />
+				  <input type="text" disabled value="root" />
+				</label>
+			  </p>
+			  <p>
+				<label>
+				  Host password:<br />
+				  <input type="password" id="pwd" autofocus />
+				</label>
+			  </p>
+			  <p>
+				<button class="btn" type="submit">
+				  Connect
+				</button>
+			  </p>
+			</fieldset>
+		  </form>
+		  <form
+			id="config"
+			onsubmit="event.preventDefault(); deploy()"
+			style="display: none"
+		  >
+			<fieldset>
+			  <p>
+				<label>
+				  Deploy on<br />
+				  <select id="srs"></select>
+				</label>
+			  </p>
+			  <p>
+				<label>
+				  IP (leave empty to use DHCP)<br />
```

Comment:
That would be great to group more the network forms because it's using a lot of space, plus we must add registration too somewhere. Another trick would be to use CIDR notation to remove on form (netmask) adding just a slight complexity (transforming CIDR to netmask)